x-pack/plugins/lens/public/visualizations/xy/xy_config_panel/annotations_config_panel/tooltip_annotation_panel.tsx

```diff
@@ -110,11 +116,19 @@ export function TooltipSection({
     return (
       <>
         <EuiFlexItem grow={true}>
-          <EuiButton isDisabled fullWidth onClick={() => {}}>
-            {i18n.translate('xpack.lens.xyChart.annotation.tooltip.noFields', {
-              defaultMessage: 'None selected',
-            })}
-          </EuiButton>
+          <EuiPanel
+            color="subdued"
+            paddingSize="s"
+            className="lnsConfigPanelAnnotations__noFieldsPrompt"
+          >
+            <EuiTextAlign textAlign="center">
+              <EuiText size="s">
+                {i18n.translate('xpack.lens.xyChart.annotation.tooltip.noFields', {
+                  defaultMessage: 'None selected',
+                })}
+              </EuiText>
+            </EuiTextAlign>
```

Comment:
Can combine `EuiText` and `EuiTextAlign` here. Can also put color assignment here.

```suggestion
            <EuiText color="subdued" size="s" textAlign="center">
              {i18n.translate('xpack.lens.xyChart.annotation.tooltip.noFields', {
                defaultMessage: 'None selected',
              })}
            </EuiText>
```